torch/special/__init__.py

```diff
@@ -1014,6 +1015,20 @@
     {out}
 """.format(**common_args))
 
+gamma = _add_docstr(_special.special_gamma,
+                    r"""
+gamma(input, *, out=None) -> Tensor
+
+Gamma function :math:`\Gamma\left(\text{input}\right)`.
```

Comment:
I know we'll come back for the docs one day, but elaborating here would be nice

Reply:
Updated!